src/platform/src/main/java/org/geoserver/platform/resource/FileSystemResourceStore.java

```diff
@@ -496,6 +504,12 @@ public boolean renameTo(Resource dest) {
             }
             return true;
         }
+        
+      
+        @Override
+        public byte[] getContents() throws IOException {
+            return java.nio.file.Files.readAllBytes(file.toPath());
```

Comment:
(The above suggestion does assume that Resource is used to access small config files)